Update CONTRIBUTING.md (#4)
nation3/.github: CONTRIBUTING.md

When you have 2 approved reviews, merge the pull request.

## Testnet Tokens

### Goerli Eth

> We use Goreli for our testnet deployments. You can get some testnet `ETH` from the following faucets:

Reviewer: ```suggestion
We use Goerli for our testnet deployments. You can get some testnet `ETH` from the following faucets:
```

- [Goerli PoW Faucet](https://goerli-faucet.pk910.de/)

> - [Alchemy Faucet](https://goerlifaucet.com/):

Reviewer: ```suggestion
- [Alchemy Faucet](https://goerlifaucet.com/)
```

- [https://faucet.paradigm.xyz/](https://faucet.paradigm.xyz/)

### $NATION

> `$NATION` tokens can be purchased from the Goreli [Balancer Pool](https://goerli.balancer.fi/#/pool/0x6f57329d43f3de9ff39d4424576db920b55060b30002000000000000000000d7) or by asking one of the members of the developer guild in [discord](https://discord.gg/zYsf6edHBf)

Reviewer: ```suggestion
`$NATION` tokens can be purchased from the Goerli Balancer Pool at https://goerli.balancer.fi/#/trade/ether/0x333A4823466879eeF910A04D473505da62142069 or by asking one of the members of the developer guild in [Discord](https://discord.gg/nation3)
```

### $veNATION

> Once you have 2 `$NATION` tokens, head over to the staking page in the [Citizen App](https://app.nation3.org/lock)

Reviewer: ```suggestion
Once you hold `$NATION` tokens, head over to the staking page in the [Citizen App](https://app.nation3.org/lock).

Note that you will need to lock _more than_ 2 `$NATION` to get 2 `$veNATION` (since your `$veNATION` balance decreases over time.)
```

When you have 2 approved reviews, merge the pull request.

## Test Network

> We use Goreli for our testnet deployemnts deployments. You can get some testnet ETH from the following faucets:

Reviewer: ```suggestion
We use Goerli for our testnet deployments. You can get some testnet ETH from the following faucets:
```

Reviewer 2: You can tell I am not a proof reader, even now I have to concentrate to see what the differences are! Maybe I should stick to code.

When you have 2 approved reviews, merge the pull request.

## Testnet Tokens

### Goerli Eth

We use Goerli for our testnet deployments. You can get some testnet `ETH` from the following faucets:

- [Goerli PoW Faucet](https://goerli-faucet.pk910.de/)
- [Alchemy Faucet](https://goerlifaucet.com/)
- [https://faucet.paradigm.xyz/](https://faucet.paradigm.xyz/)

### $NATION

> `$NATION` tokens can be purchased from the Goerli [Balancer Pool](https://goerli.balancer.fi/#/pool/0x6f57329d43f3de9ff39d4424576db920b55060b30002000000000000000000d7) or by asking one of the members of the developer guild in [discord](https://discord.gg/zYsf6edHBf)

Reviewer: Suggested link URL correction at https://github.com/nation3/.github/pull/4#discussion_r951040221
